extensions/arc/deployment/src/main/resources/dev-ui/qwc-arc-beans.js

```diff
@@ -143,15 +143,17 @@ export class QwcArcBeans extends LitElement {
             ${level
                 ? html`<qui-badge level='${level}' small><span>${kind}</span></qui-badge>` 
                 : html`<qui-badge small><span>${kind}</span></qui-badge>`
-            }`;
+            } ${bean.canBeInactive ? html`<qui-badge level='warning' small><span>Can be inactive</span></qui-badge>` : ''}`;
```

Comment:
I wonder if it would make more sense to show the `Inactive` flag instead... I know that it would require a bit more effort because it's a runtime value but this info is "half useless". Another alternative would be to introduce a special view for "Inactive beans"...

Reply:
Yeah, that's what I also thought, but the entire _Beans_ view is based on build-time data. I'll try to figure something out... maybe we can show the build-time data immediately and then fetch the runtime status via JSON RPC and update the view?

Reply:
> Yeah, that's what I also thought, but the entire Beans view is based on build-time data. 

I know but that should not be a problem ;-).

>  maybe we can show the build-time data immediately and then fetch the runtime status via JSON RPC and update the view?

Exactly. That's what we do for example in WS next: https://github.com/quarkusio/quarkus/blob/main/extensions/websockets-next/deployment/src/main/resources/dev-ui/qwc-wsn-endpoints.js#L77-L117

Reply:
Done. It took me a while, but it turns out fairly easy. I mostly struggled with change detection (see https://lit.dev/docs/components/properties/#mutating-properties).